usr/libexec/helper-scripts/settings_echo

```diff
@@ -12,4 +12,5 @@ check_tor_bootstrap_helper_variables
 
 echo "\
 GATEWAY_IP=\"$GATEWAY_IP\"
+GATEWAY_IP6=\"$GATEWAY_IP6\"
```

Comment:
This slightly scares me because IPv6 addresses have `[`, `]`, and `:` characters in them, all of which can be treated specially by the shell in some contexts. Could we use some extra quotes to make sure nothing goes awry here? (The GATEWAY_IP variable doesn't strictly need them, but maybe we could add them there for consistency too.)

```suggestion
GATEWAY_IP="\"$GATEWAY_IP\""
GATEWAY_IP6="\"$GATEWAY_IP6\""
```